### PR TITLE
Korean translation

### PR DESCRIPTION
**Pull request by eunsucking**
Add Korean translation.
Review please @marocchino @yous 😵

### CODE REVIEW

File: exercises/each/problem.ko.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+```scss
+@each $animal in puma, sea-slug, egret, salamander {
+  .#{$animal}-icon {
+    background-image: url('/images/#{$animal}.png');
+  }
+}
+```
+
+컴파일되는 CSS입니다.
```
Comment (marocchino) [inline exercises/each/problem.ko.md]: Is 컴파일되는 a past verb?

File: exercises/for_loop/problem.ko.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# FOR 반복문
+
+`@for` 지시어는 일련의 스타일을 반복해서 출력합니다. 반복마다 카운터 변수를 사용해서 출력을 조정합니다. 지시어는 두 가지의 형태가 있습니다. `@for $var from <start> through <end>`와 `@for $var from <start> to <end>`입니다. `through`와 `to`의 차이점에 주의하세요. `$var`는 `$i`와 같이 어떤 변수도 들어갈 수 있습니다. `<start>`와 `<end>`는 항상 정수를 반환하는 Sass 스크립트입니다. `<start>`가 `<end>`보다 큰 값이 지정되면, 카운터 변수는 반복마다 증가 대신 감소합니다.
+
+`@for`문은 `$var`에 지정한 범위 내의 연속된 숫자들을 할당하여, 반복마다 `$var`의 값을 사용하여 중첩된 스타일을 지정합니다. `from ... through`의 형식의 경우 `<start>`와 `<end>`의 값을 포함하지만, `from ... to`의 형식인 경우 `<start>`와 `<end>`의 값은 포함되지 않습니다. 다음은 `through`를 사용한 경우입니다.
```
Comment (marocchino) [inline exercises/for_loop/problem.ko.md]:
```diff
-`from ... to`의 형식인 경우 `<start>`와 `<end>`의 값은 포함되지 않습니다.
+`from ... to`의 형식인 경우 `<end>`의 값은 포함되지 않습니다.
```

File: exercises/interpolation/problem.ko.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# 삽입
+
+선택자 이름을 지정하거나 선택자 안에서도 변수를 사용할 수 있습니다. `#{}`를 삽입해서 변수가 들어갈 곳을 지정하세요. 예제를 보시죠.
```
Comment (marocchino) [inline exercises/interpolation/problem.ko.md]: string interpolation is usually transrated as '문자열 보간'.

File: exercises/mixin_content/problem.ko.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# MIXIN CONTENT
+
+Mixin에 스타일 블록을 넘겨주기
+
+mixin 내부의 스타일을 설정하는 방법으로, 스타일 블록을 통째로 넘겨줄 수도 있습니다. 이 스타일 블록은 mixin 내부의 `@content` 에 설정됩니다. 이 기능은 선택자와 지시어의 구성에 관한 추상화를 가능하게 합니다.
```
Comment (marocchino) [inline exercises/mixin_content/problem.ko.md]: `@content` 에 -> `@content`에

File: exercises/variables/problem.ko.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# 변수
+
+스타일 시트에서 재활용성이 있는 정보를 변수에 저장할 수 있다면 어떨까요? 예를 들면 색깔, 글씨체 등등의 CSS 속성들이 있겠네요. Sass는 가능합니다. `$`를 사용해서 말이죠. 다음과 같이 사용할 수 있습니다. 한 번 보시죠.
```
Comment (marocchino) [inline exercises/variables/problem.ko.md]: 재활용성이 있는 정보를 -> 재활용할 값을

**Comment by marocchino** [comment]
Note that I felt a sense of discomfort in the word 선택자.

File: exercises/each/problem.ko.md (hunk 1 above)
Comment (eunsucking) [inline exercises/each/problem.ko.md]: I think It is a simple passive verb.

**Comment by eunsucking** [comment]
I also wondered how to translate the selector. Mozila Docs translate that word to '선택자' So I choose it. 
https://developer.mozilla.org/ko/docs/Web/CSS/%EC%8B%9C%EC%9E%91%ED%95%98%EA%B8%B0/%EC%84%A4%EB%A0%89%ED%84%B0

File: exercises/each/problem.ko.md (hunk 1 above)
Comment (marocchino) [inline exercises/each/problem.ko.md]: 🙆‍♂️

**Comment by marocchino** [comment]
Good enough reason. 👍

**Comment by claudiopro** [comment]
@marocchino good to go? I'll merge. @eunsucking thanks for contributing a new translation!

File: exercises/mixin_content/problem.ko.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+* html #logo {
+  background-image: url(/logo.gif);
+}
+```
+
+주의: `@content` 지시어가 한 번 이상 혹은 반복문 안에서 사용되는 경우, 스타일 블록이 각각의 호출에 중복되어 나타납니다.
+
+# 과제
+
+이전 과제에서 만든 `border` mixin에 `@content` 지시어를 추가하세요. 그리고 이전 과제에서 만든 `img` 선택자에서 border를 `@include` 할 때, `@content`에 추가될 내용으로 `border-style` 스타일을 `solid`값으로 설정하세요.
```
Comment (yous) [inline exercises/mixin_content/problem.ko.md]: \`border\` mixin에 → \`border-thickness\` mixin에

File: exercises/operators/problem.ko.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  width: 600px / 960px * 100%;
+}
+
+aside[role="complimentary"] {
+  float: right;
+  width: 300px / 960px * 100%;
+}
+```
+
+960px기준의 간단한 가변성 그리드를 만들어 봤습니다. pixel값을 받아서 백분율을 구하는 식입니다. 아주 간단하죠? 생성되는 CSS는 다음과 같습니다.
```
Comment (yous) [inline exercises/operators/problem.ko.md]: 960px 기준의

File: exercises/while_loop/problem.ko.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+# WHILE 반복문
+
+`@while` 지시어는 Sass 스크립트 표현을 사용하여, 중첩 형식의 스타일을 반복적으로 출력합니다. 조건문이 거짓이 될 때 까지요. `@for`문 보다 더 복잡한 형식의 반복문을 작성할 수 있습니다. 그렇게 많이 필요로 하는 장면은 없지만요. 예를 보시죠.
```
Comment (yous) [inline exercises/while_loop/problem.ko.md]: 때 까지요 → 때까지요

File: footer.ko.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+---
+## 힌트
+
+Sass(SCSS) 스타일 시트를 만들려면 `.scss` 확장자로 끝나는 파일을 만드세요. 그리고 과제를 모두 완료했다면 다음 명령어를 실행하세요.
+
+```sh
+$ {appname} verify stylesheet.scss
+```
+
+명령어가 실행되면 현재 만든 스타일 시트가 검토되며, 성공적이라면 'completed' 가 표시됩니다.
```
Comment (yous) [inline footer.ko.md]: 'completed'가

**Comment by yous** [comment]
Oops, I was bit late.

**Comment by eunsucking** [comment]
@yous Thanks for review. 😄 I make another pull request.

**Pull request merged**